tests/pkg/test_extern/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+#!/bin/bash
+
+rm -rf .lake/build
+
+# Function to process the output
+verify_output() {
+    awk '/error: stdout:/, /error: external command/' | sed '/error: external command/d'
+}
```

Comment:
On windows we still get
```
2023-11-28T02:45:09.3711308Z < error: stdout:
2023-11-28T02:45:09.3737497Z < .\.\.\TestExtern.lean:5:0: error: native implementation did not agree with reference implementation!
2023-11-28T02:45:09.3739223Z < Compare the outputs of:
2023-11-28T02:45:09.3739977Z < #eval Nat.not_add 4 5
2023-11-28T02:45:09.3741896Z <  and
2023-11-28T02:45:09.3754905Z < #eval match 4, 5 with
2023-11-28T02:45:09.3760717Z < | x, x_1 => 0
2023-11-28T02:45:09.3763062Z ---
2023-11-28T02:45:09.3780816Z > error: stdout:
2023-11-28T02:45:09.3784341Z > ./././TestExtern.lean:5:0: error: native implementation did not agree with reference implementation!
2023-11-28T02:45:09.3785391Z > Compare the outputs of:
2023-11-28T02:45:09.3785950Z > #eval Nat.not_add 4 5
2023-11-28T02:45:09.3786409Z >  and
2023-11-28T02:45:09.3786835Z > #eval match 4, 5 with
2023-11-28T02:45:09.3787300Z > | x, x_1 => 0
```
is that something that the sed should fix?
(Maybe add a comment why the output has to be mangled)

Reply:
Normalized, and added apology for the ad-hoc mangling.